### ISSUE TITLE
certificate is expired

### ISSUE THREAD

**Issue by batdevis** [shioyama/mobility#506]
```
» gem cert --add <(curl -Ls https://raw.github.com/shioyama/mobility/master/certs/shioyama.pem)
Added '/CN=chris/DC=dejimsta/DC=com'
» gem install mobility -P MediumSecurity
ERROR:  While executing gem ... (Gem::Security::Exception)
    certificate /CN=chris/DC=dejimsta/DC=com not valid after 2021-02-10 01:02:35 UTC
```

**Comment by shioyama** [comment]
Oh rats, thanks for the heads up

**Comment by shioyama** [comment]
Ok released 1.1.1, should work with that (you'll need to download the cert again, it was updated).
